tests/components/device_sun_light_trigger/test_init.py

```diff
@@ -176,6 +176,8 @@ async def test_lights_turn_on_when_coming_home_after_sun_set_person(hass, scanne
             {"person": [{"id": "me", "name": "Me", "device_trackers": [device_1]}]},
         )
 
+        assert await async_setup_component(hass, "group", {})
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6259301)